CHANGES.md

```diff
@@ -89,6 +89,10 @@ OpenSSL 3.4
 
    *Craig Lorentzen*
 
+ * The unimplemented librandom seed source is now removed.
+
+   *Dimitri John Ledkov*
+
```

Comment:
Is there any point in adding the CHANGES.md entry? I mean as this was always unusable anyway nobody could ever use it.

Reply:
Agreed, this isn't a change-worthy alteration.